zio-http/jvm/src/main/scala/zio/http/netty/NettyRuntime.scala

```diff
@@ -16,15 +16,17 @@
 
 package zio.http.netty
 
+import scala.annotation.unused
+
 import zio._
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 
 import io.netty.channel._
 import io.netty.util.concurrent.{Future, GenericFutureListener}
 
-private[zio] trait NettyRuntime { self =>
+private[zio] final class NettyRuntime(zRuntime: Runtime[Any]) {
 
-  def runtime(ctx: ChannelHandlerContext): Runtime[Any]
+  def runtime(@unused ctx: ChannelHandlerContext): Runtime[Any] = zRuntime
```

Comment:
Similarly, I was tempted to remove the `ctx: ChannelHandlerContext` argument from here. Do we know if there was some use for it, like somehow incorporating it into the zio Runtime?

Reply:
Remove 👍